packages/fxa-content-server/app/scripts/templates/partial/coppa-age-input.mustache

```diff
@@ -1,4 +1,7 @@
-<div id="coppa" class="input-row input-row-age mb-8">
-  <input type="number" pattern="\d*" id="age" placeholder="{{#t}}How old are you?{{/t}}" min="0" max="130" value="{{ age }}" {{#required}}required{{/required}} />
-  <p>{{#t}}You must be at least 13 years old to create an account.{{/t}}</p>
+<div id="coppa" class="mb-8 tooltip-container">
+  <label for="age" id="age-label" class="sr-only">{{#t}}How old are you? You must be at least 13 years old to create an account.{{/t}}</label>
+  <input type="number" pattern="\d*" id="age" class="input-text no-step" placeholder="{{#t}}How old are you?{{/t}}" aria-labelledby="age-label" min="0" max="130" value="{{ age }}" {{#required}}required{{/required}} />
+  <!-- If you update this string you must update it in the label as well.
+  This is hidden from screenreaders in favor of reading it before users enter their age. -->
+  <p aria-hidden="true" id="age-explainer" class="text-grey-500 text-left text-xs mt-3">{{#t}}You must be at least 13 years old to create an account.{{/t}}</p>
```

Comment:
Q: should we use `text-left` or should these all be `text-start` everywhere?

```sh
# from current "main" branch
git grep -EPn "text-(left|right)" packages | grep -v "/.storybook/"

packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx:101:      <td className="text-left pl-8">
packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx:560:                  <th className="text-left">Event</th>
packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx:561:                  <th className="text-left">Timestamp</th>
packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx:593:                  <th className="text-left">Event</th>
packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx:594:                  <th className="text-left">Timestamp</th>
packages/fxa-admin-panel/src/components/Permissions/index.tsx:58:          <th className="text-left pl-4">Feature</th>
packages/fxa-admin-panel/src/styles/tailwind.css:14:  @apply font-medium text-violet-900 pl-4 pr-8 text-left;
packages/fxa-content-server/app/scripts/templates/choose_what_to_sync.mustache:12:      <div class="flex flex-wrap ltr:text-left rtl:text-right ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6">
packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache:58:      <div class="flex flex-wrap ltr:text-left rtl:text-right ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6">
packages/fxa-payments-server/src/components/fields/index.tsx:99:            className="font-medium text-sm text-grey-400 block mb-2 text-left"
packages/fxa-settings/src/components/ConnectAnotherDevicePromo/index.tsx:27:      <div className="flex flex-col flex-1 text-center mobileLandscape:text-left mobileLandscape:rtl:text-right">
packages/fxa-settings/src/components/ConnectedServices/index.tsx:227:        <div className="mt-5 text-center mobileLandscape:text-left mobileLandscape:rtl:text-right">
packages/fxa-settings/src/components/ConnectedServices/index.tsx:302:                <ul className="my-4 ltr:text-left rtl:text-right">
packages/fxa-settings/src/components/Nav/index.tsx:30:      <ul className="px-6 py-7 tablet:px-8 desktop:p-0 mobileLandscape:mt-8 ltr:text-left rtl:text-right">
packages/fxa-settings/src/styles/switch.css:24:    @apply text-left border-transparent;
packages/fxa-settings/src/styles/switch.css:27:      @apply text-right;
packages/fxa-settings/src/styles/switch.css:68:    @apply text-right border-grey-200;
packages/fxa-settings/src/styles/switch.css:71:      @apply text-left;
```

Reply:
@pdehaan The admin-panel ones don't matter since it's not localized. I updated the content-server ones here, thanks for catching the `#age-explainer` one. I'm actually going to revert my other two changes fixing these in Settings components because I think we can use a 1-2 pointer maintenance item to take a look at everything in Settings and that one piece in Payments. 👍 I'll file it, I was just about to file a new Tailwind maintenance epic.

Reply:
Filed: https://mozilla-hub.atlassian.net/browse/FXA-5738